tests/test_challenges.py

```diff
@@ -213,3 +214,87 @@ def test_display_participant_and_host_challenge_lists(self):
         result = runner.invoke(challenges, ['--participant', '--host'])
         response = result.output
         assert response == self.output
+
+
+class TestChallengePhases:
+
+    def setup(self):
+        challenge_phase_list_json = json.loads(challenge_response.challenge_phase_list)
+        challenge_phase_details_json = json.loads(challenge_response.challenge_phase_details)
+
+        url = "{}{}"
+        responses.add(responses.GET, url.format(API_HOST_URL, URLS.challenge_phase_list.value).format('10'),
+                      json=challenge_phase_list_json, status=200)
+
+        responses.add(responses.GET, url.format(API_HOST_URL, URLS.challenge_phase_detail.value).format('10', '20'),
+                      json=challenge_phase_details_json, status=200)
+        self.phases = challenge_phase_list_json['results']
+        self.phase = challenge_phase_details_json
+
+    @responses.activate
+    def test_challenge_phase_lists(self):
+
+        self.output = ""
+
+        for phase in self.phases:
+            br = ("--------------------------------"
+                  "----------------------------------")
+
+            phase_title = "\n{}".format(phase["name"])
+            challenge_id = "Challenge ID: {}".format(str(phase["challenge"]))
+            phase_id = "Phase ID: {}\n\n".format(str(phase["id"]))
+
+            title = "{} {} {}".format(phase_title, challenge_id, phase_id)
+
+            description = "{}\n\n".format(phase["description"])
+            phase = "{}{}{}\n".format(title, description, br)
+
+            self.output = self.output + phase
+
+        runner = CliRunner()
+        result = runner.invoke(challenge, ['10', 'phases'])
+        response = result.output
+        assert response == self.output
+
+    @responses.activate
+    def test_challenge_phase_details(self):
```

Comment:
@isht3 Please make the function names consistent by prepending `test_` with the actual function for which the test is written. It will help us in future in detecting the test case when the code base will increase.